runtime-sdk/src/modules/consensus_accounts/mod.rs

```diff
@@ -8,18 +8,19 @@ use once_cell::sync::Lazy;
 use thiserror::Error;
 
 use oasis_core_runtime::consensus::staking::Account as ConsensusAccount;
+use oasis_runtime_sdk_macros::{handler, sdk_derive};
```

Comment:
The `consensus_accounts` module has one of the more complex MessageHandler impls. I modified it to use the new derive macro to serve as an example. Once the rest of the PR is reviews, I'll update the other modules too.